### ISSUE TITLE
feat: add alert when a commit is made to main in the SOAP gateway

### ISSUE THREAD

**Issue by SantiagoGaonaC** [hawks-atlanta/proxy-net#23]
A functionality needs to be made that allows the console or some other implementation to notify that there is a big change in the gateway in order to update the SOAP interfaces with dotnet/references.

**Comment by SantiagoGaonaC** [comment]
Hi @shoriwe , what do you think about implementing this, in the proxy console to notify of a new gateway update, this in order to avoid interface errors.

**Comment by shoriwe** [comment]
Good idea, this way we can quickly detect problems related to inconsistencies between different versions
